scanpipe/api/views.py

```diff
@@ -91,11 +89,10 @@ def results_download(self, request, *args, **kwargs):
     @action(detail=False)
     def pipelines(self, request, *args, **kwargs):
         data = {}
-        for location, name in scanpipe_app_config.pipelines:
+        for name, pipeline_class in scanpipe_app_config.pipelines.items():
             data[name] = {
-                "location": location,
-                "description": get_pipeline_class(location).get_doc(),
-                "steps": get_pipeline_graph(location),
+                "description": pipeline_class.get_doc(),
```

Comment:
Is this the structure returned by the API? and if so does it have this shape?
```
"name" : {
    "description": "lorem ipsum.....",
    "steps":  [.....]
}
```


If yes, I wonder if have a pipeline name as a mapping key make sense as this is not our ABC data API convention to always use names/value pairs for data and avoid to have values as keys, e.g. something like:

```
[{
    "name" :  "some name",
    "description": "lorem ipsum.....",
    "steps":  [.....]
}]
```